packages/test/service-load-test/src/loadTestDataStore.ts

```diff
@@ -93,10 +84,9 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
         };
         t = setTimeout(printProgress, config.testConfig.progressIntervalMs);
 
-        const clientSendCount = config.testConfig.totalSendCount / config.testConfig.numClients;
         const opsPerCycle = config.testConfig.opRatePerMin * cycleMs / 60000;
         const opsGapMs = cycleMs / opsPerCycle;
-        while (this.sentCount < clientSendCount) {
+        while (Date.now() - startTimeMs < config.leaveDelayMs) {
```

Comment:
i don't think we should keep track of the time in the client, we should just check if the this.runtime.disposed and exit the loop if we are disposed

Reply:
I'm pretty sure I'm not following this correctly... but are you saying that rather than `await stressTest.run(runConfig);` [below](https://github.com/microsoft/FluidFramework/pull/5454/files#diff-1c781eff73139fbfb1fb8795f65d2430ec4b50c8c620005fba37edfbb43870e9R203), you want to `await Promise.race([stressTest.run(runConfig), new Promise(r => setTimeout(r, runConfig.leaveDelayMs))]);` (in other words, move the temporary termination of activity from loadTestDataStore.ts to nodeStressTest.ts, and restore the final termination to be based on a number of ops sent -- counted in a dds -- rather than a number of join/leave cycles)?

Reply:
I think it's yes to the `Promise.race` in `nodeStressTest.ts`, and ambivalent on the final termination condition.